cloud_test.sh

```diff
@@ -1,14 +1,19 @@
 #!/bin/bash 
 set -xe
 
-test_path="$1"
-
+build_dir_outputs="$1"
+test_apk_path="$2"
 gcloud firebase test android models list
 gcloud firebase test android run --type instrumentation \
     --app app/build/outputs/apk/full/debug/app-full-debug.apk \
-    --test "$test_path" \
+    --test "${build_dir_outputs}/${test_apk_path}" \
     --device model=hammerhead,version=21,locale=en,orientation=portrait  \
-    --device model=athene,version=23,locale=en,orientation=landscape \
+    --device model=hammerhead,version=23,locale=en,orientation=landscape \
     --device model=sailfish,version=26,locale=en,orientation=portrait \
     --device model=sailfish,version=28,locale=en,orientation=portrait \
-    --timeout 20m
\ No newline at end of file
+    --environment-variables coverage=true,coverageFile="/sdcard/coverage.ec" \
+    --directories-to-pull /sdcard \
+    --timeout 20m
+
+covfile_path=`gsutil ls gs://test-lab-r47d1tyt8h0hm-iku3c1i8kjrux | tail -1`
```

Comment:
Use `tail -1` has a potential  bug. If there are other tasks waiting in the firebase job queue, it will start a new job immediately after current job and `tail -1` will get this new job path.

Reply:
yeah that make sense. we'd probably need to create a permanent folder in the bucket for all coverage files.